### ISSUE TITLE
New Feature set_gql/3

### ISSUE THREAD

**Issue by davidbrahiam** [tinfoil/wormwood#18]
Hi guys i am starting to use this lib in my project and i would like know if its possible to add to the lib this function/method

```elixir
defmacro set_gql(query_name, schema, query_string) when is_atom(query_name) do
    quote do
      document = GQLLoader.load_string!(unquote(query_string))
      Module.put_attribute(unquote(__CALLER__.module), :_wormwood_gql_schema, unquote(schema))
      Module.register_attribute(unquote(__CALLER__.module), unquote(query_name), persist: true)
      Module.put_attribute(unquote(__CALLER__.module), unquote(query_name), document)
    end
end
```
its basically the same as the ```load_gql/3``` but using strings instead of file_path
I put this in my project and it works pretty well.

**Comment by uhrobots** [comment]
Great! If you like, open up a PR and we can merge it in!

**Comment by davidbrahiam** [comment]
I am trying to push the branch to create the PR but i am getting permission denied

```
davidbrahiam@MacBook-Pro wormwood % git push origin feature/set_gql_3

ERROR: Permission to tinfoil/wormwood.git denied to davidbrahiam.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
I tried before with https and didn't work i added a ssh and same.
Probably i am doing the wrong procedure. Could you give a hand about it?

**Comment by davidbrahiam** [comment]
As suggested i opened a PR to merge the feature

**Comment by uhrobots** [comment]
I've merged your PR! #16 
I will also prep a hex release soon
